src/mongocxx/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ message ("mongocxx version: ${MONGOCXX_VERSION}")
 set(MONGOCXX_INLINE_NAMESPACE "v${MONGOCXX_ABI_VERSION}")
 set(MONGOCXX_HEADER_INSTALL_DIR "${CMAKE_INSTALL_INCLUDEDIR}/mongocxx/${MONGOCXX_INLINE_NAMESPACE}" CACHE INTERNAL "")
 
-set(LIBMONGOC_REQUIRED_VERSION 1.15.0)
+set(LIBMONGOC_REQUIRED_VERSION 1.17.0)
```

Comment:
This is the only non-test change.

Reply:
I actually already needed to update my C driver installation, so seems good to make it "official" through the cmake